testing/symbols/verify_exported.dart

```diff
@@ -110,8 +110,93 @@ int _checkAndroid(String outPath, String nmPath, Iterable<String> builds) {
       'JNI_OnLoad': 'T',
       '_binary_icudtl_dat_size': 'A',
       '_binary_icudtl_dat_start': 'D',
+      // TODO(fxb/47943): Remove these once Clang lld does not expose them.
+      // arm
+      '__adddf3': 'T',
+      '__addsf3': 'T',
+      '__aeabi_cdcmpeq': 'T',
+      '__aeabi_cdcmple': 'T',
+      '__aeabi_cdrcmple': 'T',
+      '__aeabi_d2lz': 'T',
+      '__aeabi_d2uiz': 'T',
+      '__aeabi_d2ulz': 'T',
+      '__aeabi_dadd': 'T',
+      '__aeabi_dcmpeq': 'T',
+      '__aeabi_dcmpge': 'T',
+      '__aeabi_dcmpgt': 'T',
+      '__aeabi_dcmple': 'T',
+      '__aeabi_dcmplt': 'T',
+      '__aeabi_ddiv': 'T',
+      '__aeabi_dmul': 'T',
+      '__aeabi_drsub': 'T',
+      '__aeabi_dsub': 'T',
+      '__aeabi_f2d': 'T',
+      '__aeabi_fadd': 'T',
+      '__aeabi_frsub': 'T',
+      '__aeabi_fsub': 'T',
+      '__aeabi_i2d': 'T',
+      '__aeabi_i2f': 'T',
+      '__aeabi_l2d': 'T',
+      '__aeabi_l2f': 'T',
+      '__aeabi_lasr': 'T',
+      '__aeabi_ldivmod': 'T',
+      '__aeabi_llsl': 'T',
+      '__aeabi_llsr': 'T',
+      '__aeabi_ui2d': 'T',
+      '__aeabi_ui2f': 'T',
+      '__aeabi_uidiv': 'T',
+      '__aeabi_uidivmod': 'T',
+      '__aeabi_ul2d': 'T',
+      '__aeabi_ul2f': 'T',
+      '__aeabi_uldivmod': 'T',
+      '__ashldi3': 'T',
+      '__ashrdi3': 'T',
+      '__cmpdf2': 'T',
+      '__divdf3': 'T',
+      '__divdi3': 'T',
+      '__eqdf2': 'T',
+      '__extendsfdf2': 'T',
+      '__fixdfdi': 'T',
+      '__fixunsdfdi': 'T',
+      '__fixunsdfsi': 'T',
+      '__floatdidf': 'T',
+      '__floatdisf': 'T',
+      '__floatsidf': 'T',
+      '__floatsisf': 'T',
+      '__floatundidf': 'T',
+      '__floatundisf': 'T',
+      '__floatunsidf': 'T',
+      '__floatunsisf': 'T',
+      '__gedf2': 'T',
+      '__gnu_ldivmod_helper': 'T',
+      '__gnu_uldivmod_helper': 'T',
+      '__gtdf2': 'T',
+      '__ledf2': 'T',
+      '__lshrdi3': 'T',
+      '__ltdf2': 'T',
+      '__muldf3': 'T',
+      '__nedf2': 'T',
+      '__subdf3': 'T',
+      '__subsf3': 'T',
+      '__udivdi3': 'T',
+      '__udivsi3': 'T',
+      // arm64
+      '__clz_tab': 'R',
+      '__udivti3': 'T',
+      // arm64 && x64
+      '__emutls_get_address': 'T',
+      '__emutls_register_common': 'T',
+      // jit x86
+      '__moddi3': 'T',
+      '__umoddi3': 'T',
```

Comment:
The new symbols to whitelist.

I manually checked x64, that's fine with the current list.

Reply:
This issues has been addressed in upstream by https://reviews.llvm.org/D77280